doc/README.md

```diff
@@ -53,3 +53,35 @@ You can also check out the
 [awesome flame repository](https://github.com/flame-engine/awesome-flame#articles--tutorials),
 It contains quite a lot of good tutorials and articles written by the community to get you started
 with Flame.
+
+
+## Outside of the scope of the engine
+
+Games sometimes require complex feature sets depending on what the game is all about. And some of
+those feature sets are outside of the scope of the Flame Engine ecosystem, in this section you can find
+them, and also some recommendations of packages/services that can be used:
+
+## Multiplayer (netcode)
```

Comment:
Since this, and the rest, are subsections to "Outside of the scope of the engine" they should have `###` instead of `##`.